python/ray/data/dataset.py

```diff
@@ -324,6 +324,12 @@ def map_batches(
 
         This is a blocking operation.
 
+        .. tip::
+
+            For some standardized operations like imputing, encoding or normalization,
+            one may find directly using ``ray.data.preprocessors.Preprocessor`` to be
```

Comment:
We should make sure that this is a cross-reference (single backticks), so users can jump to it in the online docs.

Reply:
Thanks! Changed to 
:py:class:`~ray.data.preprocessors.Preprocessor`